task_api/app/controllers/api/labels_controller.rb

```diff
@@ -34,7 +35,15 @@ def bulk_create
 
   private
 
+  def validate_task_exists
+    if !Task.exists?(id: params[:task_id])
+      render json: { message: 'Specified Task Not Found', data: nil }, status: :not_found
+    end
+  end
+
   def set_label
+    return render json: { message: 'Label name is required', data: nil }, status: :bad_request if !params[:name]
```

Comment:
unless の方がいいです。
`params[:name]` が空の文字列の場合は、OK？でいいんですかね？

```suggestion
    return render json: { message: 'Label name is required', data: nil }, status: :bad_request unless params[:name]
```

Reply:
Rubyだと空文字列もTrueと評価されるんですね．．．
空文字列の場合もnilの場合と同様に処理したいので，条件式を変更致します